packages/react-dom/src/__tests__/ReactDOMFloat-test.js

```diff
@@ -5230,6 +5230,65 @@ body {
     );
   });
 
+  it('can update link tags from Resource to instance and back', async () => {
```

Comment:
Where is the `and back` part in this test? I thought the first render renders a link tag to Resource, the second render to instance but I'm not seeing a third render.

Reply:
I think this is meant be read as "can update link tags from Resource to instance and vice versa"?

Reply:
Yeah, that was my original poorly worded intent but I think it's probably worth another round trip give how finicky the codepaths here are

Reply:
Thanks for pointing this out btw, found a bug when going "back again" :)